MediaBrowser.Providers/People/TvdbPersonImageProvider.cs

```diff
@@ -99,13 +93,15 @@ private RemoteImageInfo GetImageFromSeriesData(Series series, string personName,
 
         private RemoteImageInfo GetImageInfo(string xmlFile, string personName, CancellationToken cancellationToken)
         {
-            var settings = _xmlSettings.Create(false);
-
-            settings.CheckCharacters = false;
-            settings.IgnoreProcessingInstructions = true;
-            settings.IgnoreComments = true;
+            var settings = new XmlReaderSettings()
```

Comment:
I see this code being copied over and over, can we have it in one place like a static function?..